<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Modelagem ArboBA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 37, 41);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameInicial">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>1021</width>
      <height>841</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="sairButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>550</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(108, 117, 125);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciarButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(108, 117, 125);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sobreButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(108, 117, 125);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Sobre</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameSobre">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>841</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>120</y>
       <width>841</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Helvetica','Arial','sans-serif','Apple Color Emoji','Segoe UI Emoji'; font-size:20pt; color:#000000;&quot;&gt;Esta aplicação tem como objetivo a conclusão do Projeto de Iniciação Científica Modelagem ArboBA. ...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmarButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>670</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(27, 135, 85);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
   </widget>
   <zorder>frameSobre</zorder>
   <zorder>frameInicial</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
